packer/packer.go

```diff
@@ -101,6 +101,10 @@ func permSize(devsize uint64) uint32 {
 	return permSize
 }
 
+func PermSizeInKB(devsize uint64) uint32 {
+	return permSize(devsize) / 2
```

Comment:
The `/ 2` is not immediately obvious to understand. How about:

```go
func PermSizeInKB(devsize uint64) uint32 {
	permSizeLBA := permSize(devsize)
	permSizeBytes := permSizeLBA * 512
	return permSizeBytes / 1024
}
```

Reply:
Yes, the `/ 2` isn't great.  Thanks for the suggestion!